src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.test.tsx

```diff
@@ -28,10 +29,19 @@ import ViewInApp from 'src/shared/components/view-in-app/ViewInApp';
 import { createGene } from 'tests/fixtures/entity-viewer/gene';
 import { createTranscript } from 'tests/fixtures/entity-viewer/transcript';
 
+jest.mock('@apollo/client', () => ({
+  gql: jest.fn(),
+  useQuery: jest.fn()
+}));
+
 jest.mock('src/shared/components/view-in-app/ViewInApp', () => () => (
   <div>ViewInApp</div>
 ));
 
+jest.mock('src/shared/components/instant-download', () => ({
+  InstantDownloadTranscript: () => <div>ViewInApp</div>
```

Comment:
The mock text should say something different.

Reply:
✅